### PR TITLE
Fix/condo/doma 4224/fix stead support

### PR DESCRIPTION
**Pull request by Kanol**
Problem:
Some Properties have address with "участок/уч." instead of "дом/д.". It cause visual problems on Properties page. Also we can't add property like this
Solution:
Add stead support

### CODE REVIEW

**Review by toplenboren — COMMENTED** [review]
Whats our product position on this?

File: apps/condo/domains/property/components/BasePropertyForm/index.tsx [hunk 1]
```diff
@@ -164,7 +164,7 @@ const BasePropertyForm: React.FC<IPropertyFormProps> = (props) => {
                                             setAddressValidatorError={setAddressValidatorError}
                                             onSelect={(_, option) => {
                                                 const address = JSON.parse(option.key as string) as AddressMetaField
-                                                if (!validHouseTypes.includes(address.data.house_type_full)) {
+                                                if (!validHouseTypes.includes(address.data.house_type_full) && address.data.stead_type_full == null) {
```
Comment (toplenboren) [inline apps/condo/domains/property/components/BasePropertyForm/index.tsx:167]: Why not use get? 

Also i'd move logic with stead to different place, like with !validHouseTypes

**Comment by toplenboren** [comment]
how is this going to be used thorough the system?

**Comment by sitozzz** [comment]
Could this problem been resolved when we switch to "address service"?

File: apps/condo/domains/common/utils/helpers.ts [hunk 2]
```diff
@@ -47,6 +47,9 @@ export const getAddressDetails = (property: ObjectWithAddressInfo) => {
     const houseType = get(addressMeta, 'house_type')
     const houseName = get(addressMeta, 'house')
 
+    const steadType = get(addressMeta, 'stead_type')
```
Comment (AntonAL) [inline apps/condo/domains/common/utils/helpers.ts:50]: Could you please add comments, what this is used for?

**Comment by toplenboren** [comment]
?

File: apps/condo/domains/property/components/BasePropertyForm/index.tsx (hunk 1 above)
Comment (pahaz) [reply apps/condo/domains/property/components/BasePropertyForm/index.tsx:167]: Looks something wrong here

**Pull request closed without merging**